app/js/features/buyer/actions.js

```diff
@@ -0,0 +1,7 @@
+import {SET_CONTACT, SET_OFFER_ID, SET_TRADE} from './constants';
```

Comment:
It is possible to write test for action too

Reply:
Even if for those one that might not be needed as they are really simple, up to you!